config/jobs/kubernetes/enhancements/enhancements-presubmit.yaml

```diff
@@ -1,11 +1,12 @@
 presubmits:
   kubernetes/enhancements:
-  - name: pull-enhancements-verify
-    always_run: false  # TODO(Katharine,nikhita): Turn this back on when the job works.
-    decorate: true
-    spec:
-      containers:
-      - image: golang:1.11
-        command:
-        - export PATH=$PATH:$GOPATH/bin
-        - ./hack/verify.sh
+    - name: pull-enhancements-verify
```

Comment:
@nikhita -- is the indent off here?

Reply:
My editor auto-intended this here, but afaik both are correct yaml. :)